test/hotspot/gtest/logging/test_logStream.cpp

```diff
@@ -21,13 +21,14 @@
  * questions.
  *
  */
+#include "memory/resourceArea.hpp"
```

Comment:
includes should be ordered alphabetically in general, but nothing should come before precompiled.hpp

Reply:
Fixed!